blog/02-03-2023-optimizing-gas-consumption-in-celo-smart-contracts-a-step-by-step-guide/optimizing-gas-consumption-in-celo-smart-contracts-a-step-by-step-guide.md

```diff
@@ -0,0 +1,472 @@
+---
+title: Optimizing Gas Consumption in Celo Smart Contracts: A Step-by-Step Guide

```

Comment:
Hey can you remove ":" from the title it is causing the issue while deploying

Reply:
Done.